scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -2534,13 +2534,13 @@ def validate_multihead_quantized_softmax_receipt(summary: Any) -> None:
         raise AttentionKvRouteSelectorError("multi-head quantized Softmax real-valued overclaim")
     if summary["score_scale"] != 1 or summary["score_gap_clip"] != 8:
         raise AttentionKvRouteSelectorError("multi-head quantized Softmax scaling drift")
-    if summary["profiles_checked"] != 3 or summary["head_counts_checked"] != [2, 4, 8]:
+    if summary["profiles_checked"] != 4 or summary["head_counts_checked"] != [2, 4, 8, 16]:
         raise AttentionKvRouteSelectorError("multi-head quantized Softmax profile/head-count drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)